src/components/button/index.tsx

```diff
@@ -301,7 +301,13 @@ class Button extends PureComponent<Props, ButtonState> {
       } else {
         if (Constants.isWeb) {
           return (
-            <Icon style={iconStyle} tintColor={Colors.$iconDefault} source={iconSource} testID={`${testID}.icon`}/>
+            <Icon 
+              id={`button_icon_${this.props.id}`}
```

Comment:
Where the `id` prop is coming from? 
have you declared in Button props? or in Icon component's props?